middlewares/inputValidation.js

```diff
@@ -175,5 +175,53 @@ const validate = {
       return next();
     }
   ],
+  resetforgotPassword: [
+    check('password')
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('Password is required')
+      .matches(/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[0-9a-zA-Z]/, 'i')
+      .withMessage('Password must contain at least one uppercase letter, one lowercase letter and one numeric digit')
+      .trim()
+      .isLength({ min: 8 })
+      .withMessage('Password must be at least 8 characters'),
+    (req, res, next) => {
+      const errors = validationResult(req);
+      const errorMessage = {};
+      if (!errors.isEmpty()) {
+        /* istanbul ignore next-line */
+        errors.array({ onlyFirstError: true }).forEach((error) => {
+          errorMessage[error.param] = error.msg;
+        });
+        /* istanbul ignore next-line */
```

Comment:
Who introduced this?

Reply:
@mazma1 I just saw that I introduced it and I have pushed a fix to remove it and write a test for it